### DISCUSSION TITLE
Template not applied for Parts within a Bag

### DISCUSSION THREAD

**Discussion by LMSSonos** [OrchardCMS/OrchardCore#15896] [Q&A]
I have created 2 parts, lets call them WeatherPart and NewsPart. For both of them I have created templates named `LandingPage__WeatherPart` and `LandingPage__NewsPart`. Each part has rendercode something like this:
```
{% if Model.ContentItems.size > 0 %}
            {% for value in Model.ContentItems %}
<h1> make some html</h1>
```

On the landingpage I have Bag to which I add weatherpart and newspart. The Landingpage template has following code:
```
{% for item in Model.ContentItem.Content.BagPart.ContentItems %}
{{ item | shape_render }}
{% endfor %}
```

The problem is, that the Templates for Weather and NewsPart are not applied, I tried different namings but could not find out which one the correct is. It works when they are not within the Bag but directly on the page.

**Comment by sebastienros** [comment]
`{{ item | shape_display | shape_render }}` maybe ?

**Comment by LMSSonos** [comment]
Thanks for your fast answer. I tried that but the issue is that the template itself is not loaded. I tried to add some static HTML code to it and it does not load, what should the naming be for Templates of parts within a bagpart, is it still `LandingPage__Partname` or should it be `Content__Partname` or something different?
